src/averageAges.js

```diff
@@ -15,11 +15,12 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const men = people.filter(person =>
+    person.sex === 'm'
+      && (typeof century !== 'number'
+        || Math.ceil(person.died / 100) === century));
```

Comment:
```suggestion
    person.sex === 'm' && (
      !number || Math.ceil(person.died / 100) === century)
    );
```

Reply:
Nice solution btw, but it's enough to convert `number` to opposite boolean value (there can't be 0 century anyway)